android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java

```diff
@@ -127,15 +127,13 @@ public void run() {
         });
     } 
 
-  /**
-   * Starts the SDK.
-   * @param token string The token that identifies the app, you can find
-   * it on your dashboard.
-   * @param invocationEventValues ReadableArray The events that invokes
-   * the SDK's UI.
-   */
-    @ReactMethod
-    public void start(final String token, final ReadableArray invocationEventValues) {
+    /**
+     * Initializes the SDK.
+     * @param token The token that identifies the app. You can find it on your dashboard.
+     * @param invocationEventValues The events that invoke the SDK's UI.
+     */
+    @ReactMethod
+    public void init(final String token, final ReadableArray invocationEventValues) {
```

Comment:
It think we should make it take a `ReadableMap options` argument just like the JavaScript API to prevent the number of parameters from growing and being unreadable as we add more options. What do you think?

Reply:
I think we will lose the type-safety if we went with that approach.

Reply:
@a7medev I second Youssef on that.

Reply:
@DavidMina96 @TheBuggedYRN `ReadableMap` has methods to get data with while specifying the type like `getInt`, `getBoolean`, etc; which can help in type-safety, we won't lose type-safety completely.

Reply:
@a7medev Having to implicitly cast the `Object`s inside the `ReadableMap` using the `getSomeType` would be still be less type-safe than the explicit parameters, IMO. Because, not only we will be compensating on the types, but we will have to make sure that the magic strings used for keys (e.g. `"token"`) still stay in sync. If the new architecture has a solution for both of these problems, we can go with that change by then.

Reply:
Sounds reasonable, let's make keep it as is then.